index.js

```diff
@@ -253,6 +254,14 @@ function init() {
             buildBundles(site.bundles);
             // generates the pages
             buildPages(site.pages);
+            // generated pages using glob.
+            const metaJsons = glob.sync('live-examples/**/meta.json', {});
+            for (const metaJson of metaJsons) {
+                const file = fse.readJsonSync(metaJson);
+                buildPages(file.pages);
+            }
+            
+
             // clean up
             removeJSBundles();
```

Comment:
With the above changes, it probably makes more sense to move the console statement from [1] to here:
[1] https://github.com/mdn/interactive-examples/blob/master/index.js#L136